manifests/c/ChidiWilliams/Buzz/0.8.1/ChidiWilliams.Buzz.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ChidiWilliams.Buzz
+PackageVersion: 0.8.1
+InstallerType: inno
+ReleaseDate: 2023-05-01
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/chidiwilliams/buzz/releases/download/v0.8.1/Buzz-0.8.1-windows.exe
+  InstallerSha256: 8AA290E1353708103260D4212EAB2E78D53475FD9C77442E9EC92F4DA18C0281

```

Comment:
```suggestion
- Architecture: x86
  InstallerUrl: https://github.com/chidiwilliams/buzz/releases/download/v0.8.1/Buzz-0.8.1-windows.exe
  InstallerSha256: 8AA290E1353708103260D4212EAB2E78D53475FD9C77442E9EC92F4DA18C0281
  ProductCode: '{574290A2-EF7C-4845-85F3-BFF2F011A580}_is1'
  Scope: user
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  InstallerUrl: https://github.com/chidiwilliams/buzz/releases/download/v0.8.1/Buzz-0.8.1-windows.exe
  InstallerSha256: 8AA290E1353708103260D4212EAB2E78D53475FD9C77442E9EC92F4DA18C0281
  ProductCode: '{574290A2-EF7C-4845-85F3-BFF2F011A580}_is1'
  Scope: machine
  InstallerSwitches:
    Custom: /ALLUSERS
```

To get rid of blocking prompt, we need to add scope-related InstallerSwitches
<img src=https://github.com/microsoft/winget-pkgs/assets/88161975/6b49c34a-faa0-4a1e-8268-e9f8b5d68d85 height=200>